analysed-packages/update-rc.d/version-0.8/update-rc.d-0.8.spdx.yaml

```diff
@@ -0,0 +1,384 @@
+    \ the\nWaiver shall be preserved to the maximum extent permitted taking into\n\
+    account Affirmer's express Statement of Purpose. In addition, to the\nextent the\
+    \ Waiver is so judged Affirmer hereby grants to each affected\nperson a royalty-free,\
+    \ non transferable, non sublicensable, non exclusive,\nirrevocable and unconditional\
+    \ license to exercise Affirmer's Copyright and\nRelated Rights in the Work (i)\
+    \ in all territories worldwide, (ii) for the\nmaximum duration provided by applicable\
+    \ law or treaty (including future\ntime extensions), (iii) in any current or future\
+    \ medium and for any number\nof copies, and (iv) for any purpose whatsoever, including\
+    \ without\nlimitation commercial, advertising or promotional purposes (the\n\"\
+    License\"). The License shall be deemed effective as of the date CC0 was\napplied\
+    \ by Affirmer to the Work. Should any part of the License for any\nreason be judged\
+    \ legally invalid or ineffective under applicable law, such\npartial invalidity\
+    \ or ineffectiveness shall not invalidate the remainder\nof the License, and in\
+    \ such case Affirmer hereby affirms that he or she\nwill not (i) exercise any\
+    \ of his or her remaining Copyright and Related\nRights in the Work or (ii) assert\
+    \ any associated claims and causes of\naction with respect to the Work, in either\
+    \ case contrary to Affirmer's\nexpress Statement of Purpose.\n\n4. Limitations\
+    \ and Disclaimers.\n\n a. No trademark or patent rights held by Affirmer are waived,\
+    \ abandoned,\n    surrendered, licensed or otherwise affected by this document.\n\
+    \ b. Affirmer offers the Work as-is and makes no representations or\n    warranties\
+    \ of any kind concerning the Work, express, implied,\n    statutory or otherwise,\
+    \ including without limitation warranties of\n    title, merchantability, fitness\
+    \ for a particular purpose, non\n    infringement, or the absence of latent or\
+    \ other defects, accuracy, or\n    the present or absence of errors, whether or\
+    \ not discoverable, all to\n    the greatest extent permissible under applicable\
+    \ law.\n c. Affirmer disclaims responsibility for clearing rights of other persons\n\
+    \    that may apply to the Work or any use thereof, including without\n    limitation\
+    \ any person's Copyright and Related Rights in the Work.\n    Further, Affirmer\
+    \ disclaims responsibility for obtaining any necessary\n    consents, permissions\
+    \ or other rights required for any use of the\n    Work.\n d. Affirmer understands\
+    \ and acknowledges that Creative Commons is not a\n    party to this document\
+    \ and has no duty or obligation with respect to\n    this CC0 or use of the Work.\n\
+    \ "
+  licenseId: LicenseRef-CC0-1.0
+  name: Creative Commons Zero v1.0 Universal
+- extractedText: " GNU GENERAL PUBLIC LICENSE\n\nVersion 2, June 1991\n\nCopyright\
```

Comment:
this item has as licenseId LicenseRef-CC0-1.0 but the extracted text is the text of the GPL-2.0
I assume that something went wrong during the generation of this file. Did you use https://tools.spdx.org/app/convert/
I recommend either to use the spdx online tools or the spdx java tools for generation of the other spdx formats.

Reply:
@OliverFendt Thank you very much for your thorough review! I have learned a lot what are the important points to keep eyes on. I have regenerated updated the files did the following changes:
1.  I have reported the issue regarding LicenseRef-CC0-1.0 to my colleagues. For the time being, I have manually removed those text related to this license
2. I have **regenerated** the files using the tool that I have written that uses spdx python tool **https://github.com/spdx/tools-python**. Now it does not have any issue as far as I know. I will keep an eye on it for the future (Thanks again)